src/ui/qgsnewgeopackagelayerdialogbase.ui

```diff
@@ -437,19 +437,19 @@
           </widget>
          </item>
          <item row="0" column="0">
-          <widget class="QLabel" name="mDatabaseLabel">
+          <widget class="QLabel" name="mFileNameLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
-            <string>Database</string>
+            <string>File name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
-          <widget class="QLabel" name="mTableNameLabel">
+          <widget class="QLabel" name="mLayerNameLabel">
            <property name="text">
-            <string>Table name</string>
+            <string>Layer name</string>
```

Comment:
It is possible to create plain tables too. Also all layers are in fact also tables with special columns and associated entries in the system tables.